gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagActionReminderScheduler.java

```diff
@@ -96,8 +98,8 @@ public void execute(JobExecutionContext context) {
       log.info("DagProc reminder triggered for (flowGroup: " + flowGroup + ", flowName: " + flowName
           + ", flowExecutionId: " + flowId + ", jobName: " + jobName +")");
 
-      DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowGroup, flowName, flowId, jobName,
-          dagActionType);
+      DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowGroup, flowName, flowId, jobName, dagActionType);
+      dagAction.setReminder(true);
```

Comment:
why not set this in the line above?

Reply:
yes, we can change the constructor also, but then will have to change a lot of instance creation methods.

Reply:
and of course we can have two constructors also

Reply:
let's overload the constructor and create another, that makes more sense to me